modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/raft/ZonePartitionRaftListener.java

```diff
@@ -72,6 +82,26 @@ private static class CommittedConfiguration {
         }
     }
 
+    private final FinishTxCommandHandler finishTxCommandHandler;
+
+    /** Constructor. */
+    public ZonePartitionRaftListener(
+            TxStatePartitionStorage txStatePartitionStorage,
+            TxManager txManager,
+            SafeTimeValuesTracker safeTimeTracker,
+            PendingComparableValuesTracker<Long, Void> storageIndexTracker,
+            ZonePartitionId zonePartitionId
+    ) {
+        this.safeTimeTracker = safeTimeTracker;
+        this.storageIndexTracker = storageIndexTracker;
+
+        finishTxCommandHandler = new FinishTxCommandHandler(
+                txStatePartitionStorage,
+                new TablePartitionId(zonePartitionId.zoneId(), zonePartitionId.partitionId()),
```

Comment:
Could you please clarify this? Why are you using `zoneId` here? `TablePartitionId` assumes that it should be `tableId`, right?

Reply:
In IGNITE-24343, the zone partition ID itself will be passed here; right now it's not used for the per-zone case and it looks like this now just to make the code compile. I've added a TODO to make it clear.